lib/pages.ts

```diff
@@ -26,12 +26,12 @@ export async function getPages(): Promise<PostOrPage> {
   return posts;
 }
 
-// export async function getSinglePost(postSlug: string) {
-//   return await api.posts
-//     .read({
-//       slug: postSlug
-//     })
-//     .catch(err => {
-//       console.error(err);
-//     });
-// }
\ No newline at end of file
+export async function getSinglePage(postSlug: string): Promise<any> {
```

Comment:
Do we have types for these api entities?  It would be good to add them eventually instead of `any`. If not then we can try installing [this](https://www.skypack.dev/view/@types/tryghost__content-api) and then remove our type annotation. Could probably be done as a followup task though

Reply:
I ended up creating my own inside `./lib/types/ghost-types.ts`. When I tried to import them from the tryghost library. I'll look at the library you proposed.